velox/exec/RowContainer.h

```diff
@@ -817,13 +848,25 @@ class RowContainer {
   /// invalidated. Any row erase operations will invalidate column stats.
   std::optional<RowColumn::Stats> columnStats(int32_t columnIndex) const;
 
+  uint32_t columnNullCount(int32_t columnIndex) {
+    return rowColumnsStats_[columnIndex].nullCount();
+  }
+
+  bool rowColumnsStatsMinMaxValid() const {
```

Comment:
maybe without row in the name?

Reply:
Good catch. I forgot to delete this function when dropping the use of this short cut.